ftp/1.5/modules/ROOT/pages/ftp-troubleshooting.adoc

```diff
@@ -34,6 +34,40 @@ To resolve this error:
 
 For additional information about enabling verbose logging for SDK components in your Mule application, see the xref:mule-sdk::troubleshooting.adoc[troubleshoot SDK components] documentation.
 
+[[common-throws]]
+== Understand Common Throws
+
+Here is a list of common throws messages and how to interpret them.
+
+* FTP:ACCESS_DENIED
+
+An FTP Connector operation attempted to access an FTP server path and performed an action for which the user has not been granted permission.
+
+* FTP:CONNECTIVITY
+
+The connection to the FTP server was interrupted.
+
+* FTP:FILE_ALREADY_EXISTS
+
+An attempt was made to create a file with the same location and name as an existing file.
+
+* FTP:FILE_LOCK
+
+An operation made an unsuccessful attempt to acquire a lock over a file.
```

Comment:
This one isn't clear I think. What does it mean to "acquire" a lock? Is this saying an attempt to lock a file was unsuccessful? I don't think locks can be acquired. ?